advanced/Scripts/piholeDebug.sh

```diff
@@ -33,32 +33,15 @@ cat << EOM
 ::: and optionally upload the generated log to a unique and random directory on
 ::: Termbin.com. NOTE: All log files auto-delete after 1 month and you are the only
 ::: person who is given the unique URL. Please consider where you post this link.
-::: 
+:::
 EOM
 
-
```

Comment:
Why did this leave? Because it's only fired from `pihole -d` and `pihole` is a sudo script?